pkg/process/checks/process_nix.go

```diff
@@ -89,3 +92,27 @@ func calculatePct(deltaProc, deltaTime, numCPU float64) float32 {
 	pct = math.Max(pct, 0.0)
 	return float32(pct)
 }
+
+// warnECSLinuxFargateMisconfig pidMode is currently not supported on fargate windows, see docs: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#task_definition_pidmode
+func warnECSFargateMisconfig(containers []*model.Container) {
+	if fargate.IsFargateInstance() && !isECSFargatePidModeSetToTask(containers) {
+		log.Warn(`Process collection is likely misconfigured. Please ensure your task definition has "pidMode":"task" set. See https://docs.datadoghq.com/integrations/ecs_fargate/?tab=webui#process-collection for more information.`)
```

Comment:
```suggestion
		log.Warn(`Process collection may be misconfigured. Please ensure your task definition has "pidMode":"task" set. See https://docs.datadoghq.com/integrations/ecs_fargate#process-collection for more information.`)
```

Suggestion: removed `?tab=webui` query parameter because it's not relevant here, and rephrased to be "softer" (IMO at least, feel free to reject)

Reply:
done!